configs/BROTHERHOBBYF722V3/config.h

```diff
@@ -0,0 +1,112 @@
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_MS5611
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25N02K
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC3
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PB10
+#define MOTOR4_PIN           PB11
+#define MOTOR5_PIN           PB4
+#define MOTOR6_PIN           PB5
+#define MOTOR7_PIN           PC8
+#define MOTOR8_PIN           PC9
+#define RX_PPM_PIN           PA3
+#define LED_STRIP_PIN        PA15
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PC10
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PC11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PA8
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define CAMERA_CONTROL_PIN   PB7
+#define ADC_VBAT_PIN         PC2
+#define ADC_RSSI_PIN         PC0
+#define ADC_CURR_PIN         PC1
+#define FLASH_CS_PIN         PB6
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(0,  PB6,  1,  0)\
+    TIMER_PIN_MAP(1,  PB7,  1,  0)\
+    TIMER_PIN_MAP(2,  PB8,  1,  0)\
+    TIMER_PIN_MAP(3,  PB9,  1, -1)\
+    TIMER_PIN_MAP(4,  PB0,  2,  0)\
+    TIMER_PIN_MAP(5,  PB1,  2,  0)\
+    TIMER_PIN_MAP(6,  PB5,  1,  0)\
+    TIMER_PIN_MAP(7,  PB4,  1,  0)\
+    TIMER_PIN_MAP(8,  PA8,  1,  0)\
+    TIMER_PIN_MAP(9,  PA3,  2,  0)\
+    TIMER_PIN_MAP(10, PB3,  1,  0)\
+    TIMER_PIN_MAP(11, PC9,  2,  0)
+
+#define ADC1_DMA_OPT        0
+
+#define MAG_I2C_INSTANCE                    (I2CDEV_1)
+#define BARO_I2C_INSTANCE                   (I2CDEV_1)
+// #define SERIALRX_PROVIDER                   CRSF
+#define DEFAULT_BLACKBOX_DEVICE             BLACKBOX_DEVICE_FLASH
+#define DEFAULT_CURRENT_METER_SOURCE        CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE        VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE         386
+#define BEEPER_INVERTED
+#define FLASH_SPI_INSTANCE                  SPI2
+#define MAX7456_SPI_INSTANCE                SPI2   
+#define GYRO_1_SPI_INSTANCE                 SPI1
+#define GYRO_1_ALIGN                        CW90_DEG
+#define SYSTEM_HSE_MHZ                      8       
+#define DEFAULT_DSHOT_BURST                 DSHOT_DMAR_ON
+#define SERIALRX_UART                       SERIAL_PORT_USART2
+#define MSP_DISPLAYPORT_UART                SERIAL_PORT_UART4

```

Comment:
Schematic shows this to be UART 1.

Reply:
On the schematic, UART1 is connected to the sky end, and UART2 and UART4 are receivers and electric tuners, respectively.

Reply:
@BrotherHobby Are there wire diagrams available for ports used - to verify

Reply:
@BrotherHobby If sky end means [digital] FPV output this needs to be UART1

What is electric tuners?

Reply:
> @BrotherHobby Are there wire diagrams available for ports used - to verify

Yes, it has been sent